modules/wowchemy/layouts/partials/blocks/people.html

```diff
@@ -0,0 +1,66 @@
+{{/* Wowchemy Blocks: People */}}
+{{/* Documentation: https://wowchemy.com/blocks/ */}}
+{{/* License: https://github.com/wowchemy/wowchemy-hugo-themes/blob/main/LICENSE.md */}}
+
+{{/* Initialise */}}
+{{ $page := .wcPage }}
+{{ $block := .wcBlock }}
+{{ $show_social := $block.design.show_social | default false }}
+{{ $show_interests := $block.design.show_interests | default true }}
+{{ $show_organizations := $block.design.show_organizations | default false }}
+{{ $show_role := $block.design.show_role | default true }}
+
+<div class="row justify-content-center people-widget">
+  {{ with $block.content.title }}
+  <div class="col-md-12 section-heading">
+    <h1>{{ . | markdownify | emojify }}</h1>
+    {{ if $block.content.subtitle }}<p>{{ $block.content.subtitle | markdownify | emojify }}</p>{{ end }}
+  </div>
+  {{ end }}
+
+  {{ with $block.Content }}
+  <div class="col-md-12">
+    {{ . }}
+  </div>
+  {{ end }}
+
+  {{ range $block.content.user_groups }}
+  {{ $query := where (where site.Pages "Section" "authors") ".Params.user_groups" "intersect" (slice .) }}
```

Comment:
Do you prefer if I add a `sort_by` parameter in the people block (along with a `sort_scending`), which defaults to `last_name`, in the same way as in the `collection` block? 
In this way a user can decide whether to sort them by `title`, `first_name` or `last_name`.

Reply:
Good idea, let's try align the People sort options with the similar logic from the Collection block.

Note: I forget the exact logic we have in the Collections block, but that might mean users have to input `Params.last_name` to sort by last name as opposed to just `last_name` since Hugo accesses its own page parameters differently to custom parameters.

Reply:
I set that users need to input one of `last_name` (default), `first_name` or `title`. In the template then I'll add `.Params.*`.
If something else is passed as argument, it defaults to `last_name`.